assets/src/edit-story/components/fontPicker/test/fontPicker.js

```diff
@@ -102,4 +102,50 @@ describe('Font Picker', () => {
     // layout is happening), so just expect it to be called
     expect(scrollTo).toHaveBeenCalledWith(0, expect.any(Number));
   });
+
+  it('should select the next font in the list when using the down arrow plus enter key', async () => {
+    const onChangeFn = jest.fn();
+    const { getByRole } = await getFontPicker({ onChange: onChangeFn });
+
+    const selectButton = getByRole('button');
+    fireEvent.click(selectButton);
+
+    const fontsList = getByRole('listbox');
+    expect(fontsList).toBeInTheDocument();
+
+    act(() => {
```

Comment:
Is `act` necessary here?

Reply:
It is otherwise the state change won't commit and it will be behind.